.gitignore

```diff
@@ -59,3 +59,4 @@ build-iPhoneSimulator/
 
 .docker-build-mark
 downzip-sw.js
+.npmrc
```

Comment:
In general, due to the unfortunate way in which `npm link` works currently, we are required to drop `npmrc` files that configure the directory for npm linking correctly that end up in the individual repositories.  We don't actually want these files to be committed, however, as they would confuse / break non docker usage of those apps.  I plan to write away this problem in the future (switch to yarn over npm), but this is my stopgap.